internal/infrastructure/mongo/mongodoc/user.go

```diff
@@ -58,6 +69,12 @@ func NewUser(user *user1.User) (*UserDocument, string) {
 		Lang:         user.Lang().String(),
 		Theme:        string(user.Theme()),
 		Password:     user.Password(),
+		PasswordReset: &PasswordResetDocument{
+			Token:     pwdReset.Token,
+			CreatedAt: pwdReset.CreatedAt,
+			ExpiresAt: pwdReset.ExpiresAt,
+			IsUsed:    pwdReset.IsUsed,
+		},
```

Comment:
If pwdReset is nil, PasswordReset field in UserDocument should be nil, otherwise the app will be panic